mifospay/src/main/AndroidManifest.xml

```diff
@@ -16,16 +16,11 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".merchants.ui.MerchantTransferActivity"></activity>
```

Comment:
this change is not necessary. please remove it.

Reply:
Okay

Reply:
@Bhavnaharitsa please remove the unnecssary changes from this PR

Reply:
Done